shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -169,12 +176,18 @@ public InputConnection createInputConnection(View view, EditorInfo outAttrs) {
         return lastInputConnection;
     }
 
+    @Override
+    public void disposeInputConnection() {
+        textInputChannel.done(mClient);
```

Comment:
This will result in the active Flutter text field being submitted if the user taps on e.g a webview text field.

We should probably instead unfocus the focus node in the framework.

Reply:
Would this require adding a channel method like 'unfocus' or something?

Reply:
I think so, though would want to dig a little bit more in the framework's focus code to see if we can find a better way.

Reply:
Ok after doing some homework I think what we should be doing here is add a focus node in the framework's focus tree for the platform view and get it focused when a platform view asks for focus(which will implicitly unfocus the currently focused node). This will allow for future implementation of proper focus traversal.

I'll start working on the framework PR unless you beat me to it.